tuned/daemon/controller.py

```diff
@@ -373,3 +373,48 @@ def instance_acquire_devices(self, devices, instance_name, caller = None):
 			log.info(rets)
 			return (False, rets)
 		return (True, "OK")
+
+	@exports.export("s", "(bsa(ss))")
```

Comment:
Are the outer parenthesis needed for the D-Bus type signature?

Reply:
Apparently it doesn't matter, but we do use them in other functions as well (e.g., `auto_profile`), so I added them to `instance_get_devices` too and kept them here.

Reply:
OK, thanks for the info.